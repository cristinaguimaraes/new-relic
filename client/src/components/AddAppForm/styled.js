import styled from 'styled-components';

export const Helper = styled.span`
 display: block;
 color: red;
 margin-top: 4px;
 font-family: 'Helvetica Bold';

  &:first-letter {
    text-transform: capitalize;
  }
`;

export const Input = styled.input`
font-family: 'Helvetica';
height: 48px;
min-height: 48px;
`;

export const Label = styled.label`
font-family: 'Helvetica Bold';
margin-top: 15px;
margin-bottom: 5px;
  &:first-letter {
    text-transform: capitalize;
  }
`;

export const Form = styled.form`
  width: 50%;
  display: flex;
  flex-direction: column;
  padding: 24px 16px;
  border-radius: 4px;
  margin-top: 24px;
  border: solid 1px

`;

export const ButtonWrapper = styled.div`
width: 100%;
display:flex;
justify-content:center
`;

export const Button = styled.button`
width: 80px;
margin-top: 15px;
border-radius: 6px;
height: 40px;
`;

export const Title = styled.div`
font-size: 35px;
font-family: 'Helvetica Bold';
color: #000000;
line-height: 43px;
margin-bottom: 15px;
text-align: center
`;

