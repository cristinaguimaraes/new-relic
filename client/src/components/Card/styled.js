import styled from 'styled-components';

export const Host = styled.div`
font-size: 16px;
font-family: 'Helvetica Bold';
color: #000000;
line-height: 19px;
margin-bottom: 23px;
`;

export const App = styled.div`
display: flex;
  :hover {
      cursor: pointer;
    };
 font-family: 'Helvetica Bold';
`;

export const Wrapper = styled.div`
display:flex;
width:${props => (props.layout === 'grid' ? '315px' : '780px')};
min-width: ${props => (props.layout === 'grid' ? '315px' : '780px')};
max-width: ${props => (props.layout === 'grid' ? '315px' : '780px')};
padding: 30px;
flex-direction: column;
background-color: white;
margin-top: 30px;
margin-left: ${props => ((props.layout === 'grid' && props.index % 2) ? '30px' : '0px')};
`;

export const Apdex = styled.div`
color: #4A4A4A;
opacity: 62%;
font-size: 13px;
font-family: 'Helvetica Bold';
line-height: 19px;
min-width: 22px;
`;

export const Name = styled.div`
color: #4A4A4A;
font-size: 16px;
font-family: 'Helvetica';
line-height: 19px;
margin-left: 10px
`;
