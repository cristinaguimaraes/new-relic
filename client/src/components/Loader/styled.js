import styled, { keyframes } from 'styled-components';

const rotate = keyframes`
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
    }
`;

export const Spinner = styled.div`
  opacity: 1;
  position: relative;
  transition: opacity linear 0.1s;
  animation: ${rotate} 2s ease-in-out 0s infinite;

  &::before {
    border: solid 3px #eee;
    border-bottom-color: red;
    border-radius: 50%;
    content: '';
    height: 40px;
    left: 50%;
    opacity: inherit;
    position: absolute;
    top: 50%;
    transform: translate3d(-50%, -50%, 0);
    transform-origin: center;
    width:  40px;
    will-change: transform;
  }
`;

export const Wrapper = styled.div`
padding-top: 50px;
 height: 200px;
 display: flex;
 align-items:center;
 justify-content: center
`
