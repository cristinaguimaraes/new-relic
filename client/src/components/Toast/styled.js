import styled from 'styled-components';

export const TextWrapper = styled.div`
  width: 100%;
  box-shadow: 0 2px 12px 4px rgba(0, 0, 0, 0.19);
  border-radius: 6px;
  color: white;
  justify-content: flex-start;
  flex-direction: row;
  padding: 16px;
  display: flex;
  align-items: center;
  background-color: green;
`;

export const Text = styled.span`
  margin-left: 8px
`;

export const IconWrapper = styled.div`
  display: flex;
  align-items: center;
  margin-left: auto;
`;
