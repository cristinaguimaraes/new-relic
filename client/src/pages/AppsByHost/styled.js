import styled from 'styled-components';

export const Content= styled.div`
width: 840px;
display:flex;
justify-content: center;
padding: 30px;
flex-direction: column;
background-color: #F8F8F8
`;

export const Cards= styled.div`
display:flex;
flex-wrap: wrap;
width: 840px;
`;


