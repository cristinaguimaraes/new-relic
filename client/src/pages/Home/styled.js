import styled from 'styled-components';
import { Link as ReactLink } from 'react-router-dom';

export const Content= styled.div`
width: 840px;
display:flex;
justify-content: center;
padding: 30px;
flex-direction: column;
background-color: #F8F8F8;
align-items: center
`;

export const Link = styled(ReactLink)`
font-size: 35px;
font-family: 'Helvetica Bold';
color: #000000;
line-height: 43px;
`
