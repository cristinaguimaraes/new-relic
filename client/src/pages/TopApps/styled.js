import styled from 'styled-components';

export const Content= styled.div`
width: 840px;
display:flex;
justify-content: center;
padding: 30px;
flex-direction: column;
background-color: #F8F8F8
`;

export const Header = styled.div`
font-size: 35px;
font-family: 'Helvetica Bold';
color: #000000;
line-height: 43px ;
`;


