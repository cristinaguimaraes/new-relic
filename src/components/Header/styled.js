import styled from "styled-components";

export const Title = styled.div`
  font-size: 35px;
  font-family: "Helvetica Bold";
  color: #000000;
  line-height: 43px;
`;

export const Text = styled.div`
  font-size: 18px;
  font-family: "Helvetica";
  color: #000000;
  line-height: 22px;
  margin-left: 8px;
  margin-bottom: 3px;
`;

export const Wrapper = styled.div`
  display: flex;
  align-items: flex-end;
`;

export const Icon = styled.i`
  margin-left: 30px;
  margin-bottom: 3px;
`;
