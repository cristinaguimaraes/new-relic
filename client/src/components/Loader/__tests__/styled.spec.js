import React from 'react';
import { shallow} from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { Spinner, Wrapper } from '../styled';

Enzyme.configure({ adapter: new Adapter() });

describe('Loader styled components', () => {
  test('Spinner should render properly', () => {
    expect(shallow(<Spinner />)).toMatchSnapshot();
  });

  test('Wrapper should render properly', () => {
    expect(shallow(<Wrapper />)).toMatchSnapshot();
  });

});
