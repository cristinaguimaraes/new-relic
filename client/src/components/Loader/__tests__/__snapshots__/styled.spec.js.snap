// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader styled components Spinner should render properly 1`] = `ShallowWrapper {}`;

exports[`Loader styled components Wrapper should render properly 1`] = `ShallowWrapper {}`;
